xbmc/addons/include/xbmc_pvr_types.h

```diff
@@ -96,6 +96,7 @@ extern "C" {
    * @brief special PVR_TIMER.iParentClientIndex value to indicate that a timer has no parent.
    */
   const unsigned int PVR_TIMER_NO_PARENT = 0; /*!< @brief timer has no parent; it was not scheduled by a repeating timer. */
+  const unsigned int PVR_INVALID_TIMER_ID = 0; /*!< @brief timer has no defined client id. */
```

Comment:
No, no and no!!! Sorry, but now I'm getting really exhausted. I told you again and again that 0 is no special value!